draft/jumpstarter.rst

```diff
@@ -20,175 +20,143 @@ CEP XXXX: Jumpstarter
-*decade*, and Celery is well positioned for a large restructuring that allows
-it both internally take advantage of the latest Python features, and also
-provide external integration with them as well. 
-
-With all of these mentioned, and given the chord example, again, we could
-continue to try and improve and fix things as is, or we could take a step back
-and reflect as to *why* there are so many subleties and quirks with the
-implementation. I think one of the ultimate reasons boils down to a lack of
-simplicity. In Celery, certain aspects of the codebase are responsible for
-a number of things...
-
-^ TODO: I want to elborate on this more, but I'm not sure if it's appropriate.
-I think the Actor Model and modeling the worker as an Actor System gives us a
-number of benefits, which I want to succintly summarize in the beginning and
-give a lot more detail as a part of the CEP. Specifically, I'd like to
-explain/say why modeling things as an Actor system will make Celery a lot
-easier to reason about, maintain, and add new, for example, Canvas primitives
-to (or even re-build/design them, etc. if that's something that's desired).
-There are clear advantages to keeping state internal to the worker, for
-example, adding cancel groups, and other things that make some of the more
-asynchronous parts of Celery easier to work with, test, and reason about. The
-main thing also, though, is to really separate concerns and responsibilities.
-It *seems* to me that some of the current Celery code is not very SRP, in the
-sense that it's responsible for calling code at a number of different layers
-and doing a number of complex things. With the actor system, each actor gets a
-lot more specialized and I think that makes the implementation a lot easier to
-both reason about and extend.  Composition (vs., for example, inheritance
-currently present with ``Signature`` objects for example to then create the
-canvas objects) then becomes a lot more attractive and possible.  To give an
-example, a ``chord`` then simply becomes responsible for passing a message to
-the underlying ``Task`` (s) or ``result`` (s) (to define more) that would
-essentially make them then send off another "message" (we'll call it right now)
-or messages to other components in the actor system that are then responsible
-for handling that message (think of ``on_chord_part_return`` here).
+  * It doesn't support ``async/await`` currently, and hasn't supported it from the beginning.
+  * It powers `Mopidy`_, and probably a number of other significant projects rely on it to some extent, so it wouldn't make sense to rely upon it for reasons listed above.
 
+* `Cell`_ was an earlier attempt at an actor model/framework for Celery. It wasn't very widely used and developed.
 
+  * Given reason #1 above, it makes sense to archive `Cell`_ and move forward with `Jumpstarter`_ (`comment <https://github.com/celery/jumpstarter/issues/1#issuecomment-755347761>`_).
```

Comment:
Note that Cell implemented actors **on top** of the existing Celery implementation.
You couldn't use it to implement Celery itself.